npc/quests/bard_quest.txt

```diff
@@ -1925,9 +1925,9 @@ morocc,134,111,3	script	Bard#3	2_M_BARD_ORIENT,{
 				mes "You don't need to be scared, that person is a servant of God, okay?";
 			} else if (Class == Job_Thief || Class == Job_Rogue) {
```

Comment:
(BaseJob == Job_Thief || BaseJob == Job_Rogue)

Reply:
ah that makes sense, will change later.